src/include/platform/PlatformManager.h

```diff
@@ -174,14 +176,7 @@ class PlatformManager
     friend class Internal::GenericThreadStackManagerImpl_OpenThread_LwIP;
     template <class>
     friend class Internal::GenericConfigurationManagerImpl;
-    // Parentheses used to fix clang parsing issue with these declarations
-    friend ::CHIP_ERROR(::chip::System::Platform::Eventing::PostEvent)(::chip::System::Layer & aLayer,
```

Comment:
Will WatchableEventManager take care of all kinds of chip event handling? So we will have only one event queue inside CHIP SDK, right?

Reply:
For the next step, `WatchableEventManager` will become the implementation class for the `System::Layer` interface. Ultimately the platform implementation _should_ be able to glue all CHIP events onto a platform native event system if there is one, or else provide an implementation based on lower-level primitives like `select()` if there isn't.